dts/arm64/fvp-aemv8r/fvp-aemv8r.dtsi

```diff
@@ -79,5 +79,35 @@
 			label = "UART_0";
 			clocks = <&uartclk>;
 		};
+
+		uart1: uart@9c0a0000 {
```

Comment:
Is this valid also for `fvp_base_revc_2xaemv8a`? Can you extend that as well?

Reply:
Yes, of course. I can do that.